docs/source/api_reference/distributions.rst

```diff
@@ -77,6 +77,16 @@ Integer support
     Binomial
     Poisson
 
+Continuous support - truncated distributions
+--------------------------------------------
+
+.. currentmodule:: skpro.distributions
+
+.. autosummary::
+    :toctree: auto_generated/
+    :template: class.rst
+
+    TruncatedNormal
```

Comment:
I would put it into full reals, since any float can end up in the support.

Alternatively, one could open a new category for interval support, `Beta` could also move there.

Reply:
I think for now we can move it to full reals itself as we do not have full fledged TruncatedDistribution class for other distributions.